pages/_document.js

```diff
@@ -106,6 +107,9 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           {!nojs && <script dangerouslySetInnerHTML={{ __html: `var _paq = _paq || [];` }} />}
+          {!nojs && <script dangerouslySetInnerHTML={{ __html: `try {
```

Comment:
You still need to remove this script here. That probably why «Etikette» currently is resized.

Reply:
OMG not sure how I missed that. Works now. Thanks!